source/develop/release-management/features/infra/aaa_faq.html.md

```diff
@@ -1,11 +1,14 @@
 ---
-title: AAA
+title: AAA FAQ
 category: feature
 authors: omachace, mperina
 wiki_category: Feature
 wiki_title: Features/AAA FAQ
-wiki_revision_count: 1
-wiki_last_updated: 2016-05-27
+wiki_revision_count: 2
+wiki_last_updated: 2016-06-03
+feature_name: AAA Frequently asked questions
+feature_modules: ovirt-engine-extensions-aaa-ldap
```

Comment:
I'd leave there only ovirt-engine-extensions-aaa as there could be things related to jdbc, ldap and misc extensions